src/main/java/ua/lviv/iot/parkingServer/model/ParkingSpot.kt

```diff
@@ -0,0 +1,15 @@
+package ua.lviv.iot.parkingServer.model
+
+import ua.lviv.iot.parkingServer.model.enums.ParkingSpotSize
+
+data class ParkingSpot(
+    var parkingSpotId: Long,
```

Comment:
maybe just `id` instead of `parkingSpotId`?

Reply:
✅